tests/cmd/auth/test_auth_config.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime, timezone
+from typing import Tuple
+
+import pytest
+
+from ggshield.cmd.main import cli
+
+from .utils import prepare_config
+
+
+expected_output = """[https://dashboard.gitguardian.com]
```

Comment:
style: should be `EXPECTED_OUTPUT` since it's a top-level constant. Or move it to TestAuthConfigList, since it's related to the `list` command.